templates/cisco_ios_show_environment_temperature.template

```diff
@@ -0,0 +1,32 @@
+Value SWITCH (\d+)
+Value SWITCH_TEMPERATURE (\w+)
+Value INLET_TEMPERATURE_VALUE (\d+)
+Value INLET_TEMPERATURE_STATE (\S+)
+Value INLET_YELLOW_THRESHOLD (\d+)
+Value INLET_RED_THRESHOLD (\d+)
+Value HOTSPOT_TEMPERATURE_VALUE (\d+)
+Value HOTSPOT_TEMPERATURE_STATE (\S+)
+Value HOTSPOT_YELLOW_THRESHOLD (\d+)
+Value HOTSPOT_RED_THRESHOLD (\d+)
+
+Start
+  ^Switch\s+\d -> Continue.Record
+  ^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$$ -> Inlet
+  ^\s*$$
+  ^. -> Error 
+ 
+Inlet
+  ^Inlet\s+Temperature\s+Value:\s+${INLET_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$ 
+  ^Temperature\s+State:\s+${INLET_TEMPERATURE_STATE}\s*$$
+  ^Yellow\s+Threshold\s+:\s+${INLET_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
+  ^Red\s+Threshold\s+:\s+${INLET_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Hotspot
+  ^\s*$$
+  ^. -> Error 
+ 
+Hotspot
+  ^Hotspot\s+Temperature\s+Value:\s+${HOTSPOT_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$
+  ^Temperature\s+State:\s+${HOTSPOT_TEMPERATURE_STATE}\s*$$
+  ^Yellow\s+Threshold\s+:\s+${HOTSPOT_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
+  ^Red\s+Threshold\s+:\s+${HOTSPOT_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Record Start
```

Comment:
You can remove the `Record` from the end of L30 and just leave **Start**

Reply:
Thank you.  Will do in morning and test.

Reply:
Thank you.  Will do in morning and test.

Reply:
wait.  I didn't push.  I am working on this now.  Sorry.  Give me about 30 minutes.

Reply:
Sorry for the delay.  It's fixed and pushed into the pull request